website/docs/docs/quickstarts/dbt-cloud/databricks-qs.md

```diff
@@ -0,0 +1,286 @@
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/go_to_sql.png" title="SQL space" />
+    </div>
+3. We will be setting up a SQL warehouse now.  Select **SQL Warehouses** from the left hand side console.  You will see that a default SQL Warehouse exists.  
+    <!--
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/sql_endpoints.png" title="SQL warehouses" /> 
+    </div>
+    -->
+4. Click **Start** on the Starter Warehouse.  This will take a few minutes to get the necessary resources spun up.
+
+5. Once the SQL Warehouse is up, click **New** and then **File upload** on the dropdown menu. 
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/new_file_upload_using_databricks_SQL.png" title="New File Upload Using Databricks SQL" />
+    </div>
+
+6. Let's load the Jaffle Shop Customers data first. Drop in the `jaffle_shop_customers.csv` file into the UI.
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/databricks_table_loader.png" title="Databricks Table Loader" />
+    </div>
+
+7. Update the Table Attributes at the top:
+
+    - <b>data_catalog</b> = hive_metastore
+    - <b>database</b> = default 
+    - <b>table</b> = jaffle_shop_customers
+    - Make sure that the column data types are correct. The way you can do this is by hovering over the datatype icon next to the column name. 
+        - <b>ID</b> = bigint
+        - <b>FIRST_NAME</b> = string
+        - <b>LAST_NAME</b> = string
+
+    <div style={{maxWidth: '600px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/jaffle_shop_customers_upload.png" title="Load jaffle shop customers" />
+    </div>
+
+8. Click **Create** on the bottom once you’re done. 
+
+9. Now let’s do the same for `Jaffle Shop Orders` and `Stripe Payments`. 
+
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/jaffle_shop_orders_upload.png" title="Load jaffle shop orders" />
+    </div>
+
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/stripe_payments_upload.png" title="Load stripe payments" />
+    </div>
+    
+10. Once that's done, make sure you can query the training data.  Navigate to the `SQL Editor` through the left hand menu.  This will bring you to a query editor.
+11. Ensure that you can run a `select *` from each of the tables with the following code snippets. 
+
+    ```sql
+    select * from default.jaffle_shop_customers
+    select * from default.jaffle_shop_orders
+    select * from default.stripe_payments
+    ```
+
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/query_check.png" title="Query Check" />
+    </div>
+
+12. To ensure any users who might be working on your dbt project has access to your object, run this command.
+
+    ```sql 
+    grant all privileges on schema default to users;
```

Comment:
I don't know enough to _reject_ this, but it feels like a big hammer to solve a problem. Do we want to encourage people who might not be experienced DBA's to solve their problems with `grant all privileges to all users`?